package.json

```diff
@@ -1,21 +1,31 @@
 {
-  "name": "begin-app",
+  "name": "tc39-ci",
   "version": "0.0.7",
-  "description": "Begin basic Hello World! app",
+  "description": "Publishes TC39 drafts and updates to S3",
   "scripts": {
     "lint": "eslint . --ignore-pattern node_modules --fix",
-    "start": "cross-env NODE_ENV=testing npx sandbox",
-    "test": "echo 'tests skipped' || cross-env NODE_ENV=testing PORT=6666 tape test/*-test.js ./test/**/*-test.js | faucet",
+    "start": "cross-env npx sandbox",
+    "test": "echo 'tests skipped' || cross-env PORT=6666 tape test/*-test.js ./test/**/*-test.js | faucet",
```

Comment:
```suggestion
    "test": "echo 'tests skipped' || cross-env PORT=6666 tape 'test/**/*-test.js' | faucet",
```

Reply:
Per above, I think we may need cross env to properly glob there? Not sure but we don't have any tests here anyway, so no big deal.

Reply:
i don't think cross-env has any effect on globbing?